privacyidea/lib/token.py

```diff
@@ -2885,5 +2898,12 @@ def challenge_text_replace(message, user, token_obj):
         if email is not None:
             tags["email"] = email
 
+    # If the message is for a pushtoken and the presence_answer is set, but there is no tag for placing that answer,
+    # Append the answer to the message
+    presence_answer = tags.get("presence_answer", None)
+    if presence_answer and tokentype == "push" and "{presence_answer}" not in message:
+        message += f" Please press: {presence_answer}"
```

Comment:
Do we translate the default messages? Maybe we should do that here too?

Reply:
we have not and i did not aim to do that in this pr